astroquery/esa/hubble/core.py

```diff
@@ -717,6 +746,22 @@ def get_tables(self, only_names=True, verbose=False):
         else:
             return tables
 
+    def get_status_messages(self):
```

Comment:
you may want to mention this in the changelog or in the docs of all modules having it (`get_status_messages` is mentioned in the docs for the JWST module, but not for the rest)

Reply:
It is mentioned now.